src/generate-site.ts

```diff
@@ -256,14 +259,17 @@ const hydrateHTML = ({
     .replace(/\${PAGE_CONTENT}/g, content);
 
 export const run = async (): Promise<void> => {
-  const roamUsername = getInput("roam_username");
-  const roamPassword = getInput("roam_password");
-  const roamGraph = getInput("roam_graph");
+
+  const test = process.env.NODE_ENV === "test"   
+  const roamUsername: string = test? process.env.ROAMRESEARCH_USERNAME as string : getInput("roam_username");
```

Comment:
these inputs should already be available at `INPUT_ROAM_USERNAME` environment variables

https://github.com/actions/toolkit/blob/main/packages/core/src/core.ts#L86